### ISSUE TITLE
Add link for Slack request post to Collaboration Cycle Request ticket

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54109]
## User Story
As a Governance team member, I want to be able to easily locate a VFS teams Slack post for their Kickoff, Design Intent, Midpoint Review, and Staging Review so that I can ask questions or communicate concerns to the VFS teams in a direct, quick and reliable way.

Assignee: @CherylEvans 
Peer Reviewer: @allison0034 

## Description
We often find ourselves having to communicate with the VFS team post scheduling of their Collab Cycle touchpoint and pre- meeting.  We prefer to communicate in Slack so 1) comments don't build up within the gh ticket and 2) because we get a quick response from the VFS team when communicating in Slack vs. gh. Currently, we don't have a way to easily reference the Slack post that the VFS team made. 

The purpose of this ticket is to modify the Collaboration Cycle Kickoff-Moderator Responsibilities guidance so that we have a place within the code blocks (for DI, MPR and SR) and a place within the ticket for Kickoff to easily add the link for the Slack post the VFS team member started when notifying us of kickoff or scheduling their DI, MPR or SR touchpoints.

The 2nd purpose of this ticket is to modify the Moderator responsibilities guidance for DI, MPR and SR so that there are operational steps for the Governance team member to add the Slack post in the "Schedule" sections of DI and SR, and the synch and asynch "Schedule" sections of the MPR guidance.

## Impacted Artifacts
Collaboration Cycle Moderator Responsibilities - Kickoff
Collaboration Cycle Moderator Responsibilities - Design Intent
Collaboration Cycle Moderator Responsibilities - Midpoint Review
Collaboration Cycle Moderator Responsibilities - Staging Review

## Tasks
**Collaboration Cycle Kickoff:**
- [x] Add a new step 6 to the CC Kickoff Moderator Responsibilities doc
![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/c481c01a-5b54-4467-8763-fb77cd36f12a)
- [x] New step 6 should include details for copying the Slack thread url from Slack to the CC Request ticket in the following location:
![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/701cd32e-7087-4b7f-8a9e-7a52b8b7cb6d)
 - [x] The text should read, "Kickoff Slack thread with VFS team:" with then a link to the Slack thread.

**DI, MPR and SR ticket changes:**
- [x] Modify the code blocks for DI, MPR, and SR in the CC Kickoff Moderator Responsibilities doc
- [x] Under Platform actions, remove the "Follow Moderator Responsibilities guidance for ________" as it is not necessary for the template and often is not checked
![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/c04a4469-795c-4092-b4aa-f34972594e1f)
- [x] Replace it with, "______ Slack thread with VFS team:" where "_____" is Design Intent, Midpoint Review or Staging Review

**Operational guidance:**
- [x] Add a step to the DI, MPR and SR Moderator Responsibilities docs within the "Schedule" section for the Governance team member to copy the Slack thread and paste it where applicable
- [x] There is both a synch and asynch "Schedule" for MPR, so the change will need to be made in 2 places.
- [ ] Since we likely will not go and update every open Collab Cycle Request ticket to add the lines to the code blocks, there should be a note in our operational guidance that tells our Gov team members to add the "______ Slack thread with VFS team:" and paste the link if not included in the CC Request ticket. **(We have collectively decided that implementing this change to our operational documentation is not necessary as it would add guidance that would soon need to be deleted.)**

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] Governance team members are able to easily contact the VFS team through their initial Collab Cycle Slack post

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [ ] Ticket has been moved to the Planned pipeline in the Governance team board
- [ ] Ticket as an assignee and peer reviewer

**Comment by CherylEvans** [comment]
Draft Documentation has been created:

[Collaboration Cycle Moderator Responsibilities - Kickoff](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2550890497/54109+-+Copy+of+Collaboration+Cycle+Kickoff+-+Moderator+Responsibilities)
[Collaboration Cycle Moderator Responsibilities - Design Intent](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2550923265/54109+-+Copy+of+Design+Intent+-+Moderator+Responsibilities)
[Collaboration Cycle Moderator Responsibilities - Midpoint Review](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2550923290/54109+-+Copy+of+Midpoint+Review+-+Moderator+Responsibilities)
[Collaboration Cycle Moderator Responsibilities - Staging Review](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2550923312/54109+-+Copy+of+Staging+Review+-+Moderator+Responsibilities)

**Comment by CherylEvans** [comment]
@shiragoodman I'd like to set up a meeting with you this week to discuss some of the fine details associated with these tasks.  I will reach out on Monday.

**Comment by shiragoodman** [comment]
@CherylEvans our 1:1 is at 11:am ET today. Will that work or would you like to set up a separate meeting?

**Comment by CherylEvans** [comment]
@shiragoodman Yes, the 1:1 will work to discuss.  :-)

**Comment by allison0034** [comment]
@shiragoodman @CherylEvans I made some comments in the drafts but I have a rec for this task...

>  Since we likely will not go and update every open Collab Cycle Request ticket to add the lines to the code blocks, there should be a note in our operational guidance that tells our Gov team members to add the "______ Slack thread with VFS team:" and paste the link if not included in the CC Request ticket.

I do not think we should put 2 paragraphs of directions in our documentation to retro fit these tickets. I think we either make a new ticket to update all existing GH tickets or (and honestly this is what I vote for) just make this a change we do going forward. There are a lot of reviews coming up and this is going to take time to do. We have been dealing without the Slack thread this long I think we can deal with it for another month lol.

Adding all that content to our docs for a temporary solution probably isnt the best idea.

**Comment by shiragoodman** [comment]
I'm okay with this suggestion.

**Comment by CherylEvans** [comment]
@allison0034 @shiragoodman I am also good with Allison's suggestion.  We should focus on the path forward. It won't hurt to **not** add it to all of the ones already in progress.

**Comment by CherylEvans** [comment]
@allison0034 I have implemented your recommendations.  Let me know if you see anything else that should change. :-)

**Comment by CherylEvans** [comment]
@allison0034 Thank you for that!  I will give it a look in the morning. :-)

**Comment by CherylEvans** [comment]
@allison0034 I just looked everything over and made a few grammatical changes only.  The content looks great, thank you!
"Open the Collaboration Cycle Request ticket. Under the Staging Review/Platform Actions section, check the “Meeting date/time:” box and add the meeting date and time. Check the Staging review Slack thread with the VFS team: box and add the URL link from the #vfs-platform-support Slack where the Staging review scheduling request was made. Move the ticket to the “Staging Review Scheduled” pipeline on the Platform Governance team board."
I changed all the touchpoint docs appropriately. :-)
